lib/handlers/api.js

```diff
@@ -46,16 +46,16 @@ router.post('/:domain/:environment/db', fhmbaasMiddleware.envMongoDb.getOrCreate
 router.post('/apps/:domain/:environment/:appname/migratedb', fhmbaasMiddleware.app.findOrCreateMbaasApp, middleware.createDbMiddleware, middleware.stopAppMiddleware, middleware.migrateDbMiddleware, middleware.notifyAppDbMigration('start'), function(req, res) {
   return res.json(req.createDbResult);
 });
-
-router.get('/apps/:domain/:environment/:appname/dedicated/db', function(req, res, next) {
-  mongoUtils.hasDedicatedDb(req.params, function(err, hasDedicatedDb) {
-    if (err) {
-      return next(err);
-    }
-
-    return res.status(200).send(hasDedicatedDb);
-  });
-});
+// 
+// router.get('/apps/:domain/:environment/:appname/dedicated/db', function(req, res, next) {
+//   mongoUtils.hasDedicatedDb(req.params, function(err, hasDedicatedDb) {
+//     if (err) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)